exec/java-exec/src/main/java/org/apache/drill/exec/physical/rowSet/impl/ColumnState.java

```diff
@@ -123,7 +123,7 @@ public static MapArrayColumnState build(ResultSetLoaderImpl resultSetLoader,
       // Create the map's offset vector.
 
       UInt4Vector offsetVector = new UInt4Vector(
-          BaseRepeatedValueVector.OFFSETS_FIELD,
+          MaterializedField.OFFSETS_FIELD,
```

Comment:
While it is good idea to centralize these names, not sure the `MaterializedField` is the place, since the metadata is logically a bit below the level of vector structure.

Reply:
It is not a name. It is an immutable static `MaterializedField` that can be shared between vectors. The name still belongs to `ValueVector`.